sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/KeyOperationParameters.java

```diff
@@ -11,6 +11,8 @@
  * The key operations parameters.
  */
 class KeyOperationParameters {
```

Comment:
is this class meant to be public in future ?

Reply:
Not for the moment, it's what we use to pass parameters about any operation but we have more specialized classes that have already GA'd for this like `ImportKeyOptions`, with which we populate the `KeyOperationParameters` object to send in the request.